### DISCUSSION TITLE
TabControl / TabItems resetting to first TabItem in Designer/Previewer

### DISCUSSION THREAD

**Discussion by ManuelKetisch** [AvaloniaUI/Avalonia#12546] [Q&A] [answered]
Hello everyone,

i am fairly new to Avalonia and i am currently working on a UI that is utilising a TabControl with several TabItems.
While working on the first TabItem and placing all the controls inside, everything was fine. 
As soon as i started  to work on the next TabItems, i noticed this bahavior which i do not know is intended, a limitation, or plain noobishness of myself.
Everytime i make a change to a control inside of any TabItem that is not the first TabItem, the designer is resetting the DesignPreview to the first TabItem in the TabControl. Which makes laying out all the Controls in the remaining TabItems very tedious, because for every minor change, you have to tab back into the control you are currently working on, to see the changes you made.

Long story short, is there any way to work around this behavior? 

Thanks in advance.

**Comment by kekekeks** [comment, marked as answer]
Unfortunately the previewer does a full XAML reload every time. As a workaround you can set SelectedIndex on the tab control

**Reply by ManuelKetisch** [reply]
Aye, thanks. 
Haven't thought about it. I can work with that.
